domains/faisal.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "faisal's portfolio website",
+  "repo": "https://github.com/fasih4444/Portfolio-V4",
+  "owner": {
+    "username": "fasih4444",
+    "email": "arainfaisal826@gmail.com",
+    "twitter": "fasih4444"
+  },
+  "record": {
+    "CNAME": "faisal.vercel.app"
```

Comment:
Vercel isn't supported.
It was clearly stated in the check list that Vercel isn't supported.
Try another service like GitHub Pages or [Railway](https://railway.app)

Reply:
OK Vaibhav, I'll ping you after update.

Reply:
> Vercel isn't supported. It was clearly stated in the check list that Vercel isn't supported. Try another service like GitHub Pages or [Railway](https://railway.app)

@VaibhavSys Updated Github.io URL. Kindly Review.